odl/operator/pspace_ops.py

```diff
@@ -443,7 +424,11 @@ def _call(self, x):
             [4.0, 5.0, 6.0]
         ])
         """
-        return x[self.index].copy()
+        if out is None:
+            out = x[self.index].copy()
+        else:
+            out.assign(x[self.index])
+        return out
```

Comment:
Not necessarily since the `__call__()` does it. We need a general decision on this, though.